Bio/NMR/NOEtools.py

```diff
@@ -63,8 +63,9 @@ def predictNOE(peaklist, originNuc, detectedNuc, originResNum, toResNum):
     detectedPPMCol = datamap[detectedNuc + ".P"] + 1
 
     # Make a list of the data lines involving the detected
-    if str(toResNum) in peaklist.residue_dict(detectedNuc) \
-    and str(originResNum) in peaklist.residue_dict(detectedNuc):
+    if str(toResNum) in peaklist.residue_dict(detectedNuc) and str(
+        originResNum
+    ) in peaklist.residue_dict(detectedNuc):
```

Comment:
This is an example where I don't like how black split up a long if statement conditional. But, unless you can see a neat way to avoid this (e.g. brackets on the two arguments to and?), leave it as it is.

Reply:
Added brackets a bit better; Thank you,